docs/semgrep-app/notifications.md

```diff
@@ -88,19 +88,19 @@ To enable MR comments:
 For example:
 ```yaml
 semgrep:
-  image: returntocorp/semgrep-agent:v1
+  image: returntocorp/semgrep
   script:
     # SEMGREP_APP_TOKEN can be obtained from semgrep.dev/manage/settings.
     # The SEMGREP_APP_TOKEN should be treated like a secret and not hard-coded into your code.
-    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN
+    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN TODO
   rules:
   # Scan changed files in MRs, block on new issues only (existing issues ignored)
   - if: $CI_MERGE_REQUEST_IID
   # Scan all files on default branch, block on any issues
   # - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
 
   variables:
-    SEMGREP_AGENT_DEBUG: 1
+    SEMGREP_AGENT_DEBUG: 1 TODO
```

Comment:
This can (and probably should) be removed

Reply:
Ok, @brendongo so from the original at the time of review:

```yaml
semgrep:
  image: returntocorp/semgrep
  script:
    # SEMGREP_APP_TOKEN can be obtained from semgrep.dev/manage/settings.
    # The SEMGREP_APP_TOKEN should be treated like a secret and not hard-coded into your code.
    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN TODO
  rules:
  # Scan changed files in MRs, block on new issues only (existing issues ignored)
  - if: $CI_MERGE_REQUEST_IID
  # Scan all files on default branch, block on any issues
  # - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH

  variables:
    SEMGREP_AGENT_DEBUG: 1 TODO
    # Gives Semgrep permission to post inline comments
    GITLAB_TOKEN: $PAT
```

We can make the following:

```yaml
semgrep:
  image: returntocorp/semgrep
  script:
    # SEMGREP_APP_TOKEN can be obtained from semgrep.dev/manage/settings.
    # The SEMGREP_APP_TOKEN should be treated like a secret and not hard-coded into your code.
    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN TODO
  rules:
  # Scan changed files in MRs, block on new issues only (existing issues ignored)
  - if: $CI_MERGE_REQUEST_IID
  # Scan all files on default branch, block on any issues
  # - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH

  variables:
    GITLAB_TOKEN: $PAT
```

Reply:
(Removed FYI, just let me know if the snippet is correct now!) @brendongo

Reply:
```
    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN
```
can be just
```
    - semgrep-agent
```
as long as a `SEMGREP_APP_TOKEN` env var has been set on the gitlab web UI

Reply:
```
semgrep:
  image: returntocorp/semgrep
  script:
   # semgrep ci will retrieve the SEMGREP_APP_TOKEN envvar set using gitlab web ui
    - semgrep ci
  rules:
  # Scan changed files in MRs, block on new issues only (existing issues ignored)
  - if: $CI_MERGE_REQUEST_IID
  # Scan all files on default branch, block on any issues
  # - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH

  variables:
    GITLAB_TOKEN: $PAT
```

Reply:
@brendongo done!